lib/timers.js

```diff
@@ -614,64 +651,6 @@ function unrefdHandle() {
 }
 
 
-Timeout.prototype.unref = function() {
-  if (this._handle) {
-    this._handle.unref();
-  } else if (typeof this._onTimeout === 'function') {
-    var now = TimerWrap.now();
-    if (!this._idleStart) this._idleStart = now;
-    var delay = this._idleStart + this._idleTimeout - now;
-    if (delay < 0) delay = 0;
-
-    // Prevent running cb again when unref() is called during the same cb
-    if (this._called && !this._repeat) {
-      unenroll(this);
-      return;
-    }
-
-    var handle = reuse(this);
-    if (handle) {
-      handle._list = undefined;
-    }
-
-    this._handle = handle || new TimerWrap();
-    this._handle.owner = this;
-    this._handle[kOnTimeout] = unrefdHandle;
-    this._handle.start(delay);
-    this._handle.domain = this.domain;
-    this._handle.unref();
-  }
-  return this;
-};
-
-Timeout.prototype.ref = function() {
-  if (this._handle)
-    this._handle.ref();
-  return this;
-};
-
-Timeout.prototype.close = function() {
-  this._onTimeout = null;
-  if (this._handle) {
-    // Fewer checks may be possible, but these cover everything.
```

Comment:
Did you check this in detail? If not, can you please preserve the comment?

Reply:
I don't think we can do less checks there (I followed the breadcrumb trail as much as I could) but happy to put the comment back. 

(Also, I did remove an unnecessary `this` check.)

Reply:
Ok sounds good then.